README.md

```diff
@@ -1,2 +1,2 @@
-# Welcome to Aethos
-To run an operator, please follow the instructions on our [documentation](https://docs.aethos.network). 
+# Welcome to Predicate
+To run an operator, please follow the instructions on our [documentation](https://docs.predicate.io). 
```

Comment:
Can you also update this docs link? It currently 404s

Reply:
It will link starting tomorrow